fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/rules/InComparisonToExplodeRule.java

```diff
@@ -214,7 +215,7 @@ private static List<QueryPredicate> createSimpleEqualitiesForRecordTypeValue(@No
         for (int i = 0; i < fieldValues.size(); i++) {
             final Value fieldValue = fieldValues.get(i);
             BooleanValue currentVal = (BooleanValue) new RelOpValue.EqualsFn().encapsulate(List.of(fieldValue, comparandValueChildren.get(i).getValue()));
-            Optional<QueryPredicate> currentQueryPredicate = currentVal.toQueryPredicate(null, Quantifier.current());
+            Optional<QueryPredicate> currentQueryPredicate = currentVal.toQueryPredicate(null, ImmutableSet.of(Quantifier.current()));
```

Comment:
can you create an override for these callers that only have exactly one alias? So we don't have to explicitly create `ImmutableSet.of(...)` every time.